assets/css/styles_override.css

```
@@ -1,252 +1,767 @@
+.footer-socials a:hover,
+.footer-socials a:focus-visible {
+  color: var(--openms-white);
+  background: transparent;
 }
 
 .footer-socials svg.icon {
-  width: 1.4rem;
-  height: 1.4rem;
+  width: 1.5rem;
+  height: 1.5rem;
   fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)